tests/anmykh/test_homework_1.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+
+@pytest.mark.parametrize(["item", "data_type"],
+                         [("sad", str)])
+def test_is_string(item, data_type):
+    assert isinstance(item, data_type)
```

Comment:
more readable:

```
@pytest.mark.parametrize(["item", "data_type"], [
    ("sad", str)
])
def test_is_string(item, data_type):
    assert isinstance(item, data_type)
```

Reply:
this comment relates to all parametrizes not only to this one